using_images/s2i_images/perl.adoc

```diff
@@ -109,3 +109,6 @@ command to enter the container:
 After you enter into the running container, your current directory is set to
 `/opt/app-root/src`, where the source code is located.
 endif::openshift-origin[]
+
+== Logs
+Access logs are streamed to standard out and as such they can be viewed via the link:../../cli_reference/basic_cli_operations.html#troubleshooting-and-debugging-cli-operations[oc logs] command. Error logs are stored in *_/tmp/error_log_* which can be viewed using link:../../dev_guide/executing_remote_commands.html[oc exec] to access the container.
```

Comment:
s/standard out/standard output/
I think you could also mention `oc rsh` for the container access? Just a suggestion.
Try to wrap to 80 chars as @adellape mentioned.